uk.co.compucorp.civicrm.civihrcore/CRM/CiviHRCore.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class CRM_CiviHRCore {
+  /**
+   * Get version value from info.xml. This method uses caching so it reads
+   * xml file only once during CiviCRM session.
+   *
+   * @return string
+   */
+  public static function getVersion() {
```

Comment:
Ideally yes, it would be better for this to be an instance method, but since it uses CRM_Core_BAO_Cache and CRM_Extension_Info static methods, you would still have problems to isolate the method and unit test it.

MAYBE we can use vfsStream (https://phpunit.de/manual/current/en/test-doubles.html#test-doubles.mocking-the-filesystem) to mock the filesystem so the method will always access a stub info.xml file and then we can assert that is returns the stub's version information.